### PR TITLE
Pen 13 test suite gui

### PR DESCRIPTION
**Pull request by mcdermottm**
Requesting merge for gui code into master.

Merge conflicts coming from setup.py shouldn't be a problem to resolve, none of the existing code was actually removed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,12 @@ Team Peng - CIS 4930 - Python Term Project
 
 This test suite it built to facilitate testing of student projects by teachers according to a predefined specification.
 
-This package includes three primary tools:
+This package includes four primary tools:
```
Comment (tmaiello) [inline README.md:14]: this is a good point I'll have to remember to update this for other features!

File: README.md [hunk 2]
```diff
@@ -193,10 +194,25 @@ Command to be used to compile the build (e.g., make or devenv). This is a list o
...
+## Building & Running Graphical Interface (`peng-gui`)
+
+To run the GUI from WSL2, VcXsrv should be running first.
+
+Some additional OS packages may be required (especially on stock distributions of WSL2):
+
+`sudo apt install libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-render-util0 libxcb-xfixes0 libxcb-xinerama0`
+
+Additionally, qtmake may require the following:
+
```
Comment (tmaiello) [inline README.md:206]: should we consider packaging this in some sort of wizard/script?

File: herptest/toolbox.py [hunk 3]
```diff
@@ -94,6 +94,7 @@ def emit(self, record):
             if self._add_level:
                 record.msg = "[%s] %s" % (record.levelname, record.msg)
             logging.StreamHandler.emit(self, record)
+            logging.StreamHandler.flush(self)
```
Comment (tmaiello) [inline herptest/toolbox.py:97]: good idea to add a flush to his logging

File: setup.py [hunk 4]
```diff
@@ -9,13 +9,16 @@
     author='Jeremiah Blanchard',
     author_email='jjb@eng.ufl.edu',
     description='Test suite tools for instructors',
-    install_requires=[],
```
Comment (tmaiello) [inline setup.py:12]: this might be the only merge conflict, ill deal with it

**Review by tmaiello — APPROVED** [review]
looks good to me, approved

File: README.md (hunk 2 above)
Comment (mcdermottm) [reply README.md:206]: If it was a python environment thing, maybe. As it is, it's only a one-liner to copy but it has an effect on OS packages that I think people should be explicitly aware that theyre adding

File: setup.py (hunk 4 above)
Comment (mcdermottm) [reply setup.py]: beat me to it, thanks

**Pull request merged**